### DISCUSSION TITLE
Admob not initialized, check your configuration

### DISCUSSION THREAD

**Discussion by xolatgames** [Poing-Studios/Godot-AdMob-Android-iOS#109] [Q&A]
Downloaded, installed, configured, exported, launched this plugin. And the device says to me: Admob not initialized, check your configuration

I enabled the plugin in the project settings, in autoload, enabled it in the export window. Whatever I try, writes exactly this. How to solve this problem?

![Screenshot_2022-03-02-17-56-32-221_com test test](https://user-images.githubusercontent.com/64919860/156651586-d9676412-9bfd-400a-a14a-985658b99bfe.jpg)

**Comment by gumaciel** [comment]
Weird, it must probably be some internet problem you are having, try using some VPN to see if that solves this problem

**Comment by samelat** [comment]
Try enabling "Test Europe User Consent" in AdMob plugin config, Debug->General. I had the same issue (here in Spain) and that solved It
